OpenSim/Common/MultivariatePolynomialFunction.cpp

```diff
@@ -163,57 +169,57 @@ class SimTKMultivariatePolynomial : public SimTK::Function_<T> {
                                 if (derivComponent[0] == 0) {
                                     nqNonNegative = nq[0] - 1;
                                     if (nqNonNegative < 0) nqNonNegative = 0;
-                                    T valueP = nq[0] * std::pow(x[0], nqNonNegative);
+                                    T derivP = nq[0] * std::pow(x[0], nqNonNegative);
```

Comment:
The code above looks a lot like the code at the constructor: Perhaps worth reusing it...